contracts/linear/src/epoch_actions.rs

```diff
@@ -70,16 +74,19 @@ impl LiquidStakingContract {
                 amount_to_stake.into(),
                 env::current_account_id(),
                 NO_DEPOSIT,
-                GAS_CB_VALIDATOR_STAKED,
-            ));
-
-        true
+                GAS_CB_VALIDATOR_STAKED + GAS_SYNC_BALANCE + GAS_CB_VALIDATOR_SYNC_BALANCE,
```

Comment:
can we chain the call to sync balance here and do fire-and-forgot? like:
```rust
        candidate
            .validator
            .deposit_and_stake(&mut self.validator_pool, amount_to_stake)
            .then(ext_self_action_cb::validator_staked_callback(
                candidate.validator.account_id.clone(),
                amount_to_stake.into(),
                env::current_account_id(),
                NO_DEPOSIT,
                GAS_CB_VALIDATOR_STAKED,
            ))
            .then(validator
                .sync_account_balance()
                .then(ext_self_action_cb::validator_get_account_callback(
                    validator_id,
                    env::current_account_id(),
                    NO_DEPOSIT,
                    GAS_CB_VALIDATOR_SYNC_BALANCE,
                ))
                .into()
            );

        return true;
```
there is no need to wait for the result of callback since this function is meant to always return true here.